src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,68 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String COMMA = ",";
+    private static final String DELIMITER = " ";
+    private static String resultString;
+    private static final int ONE = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
 
+        String data = readFile(fromFileName);
+        String report = createReport(data);
+        writeFile(report, toFileName);
+    }
+
+    private String readFile(String fromFileName) {
+        StringBuilder builder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            while (bufferedReader.ready()) {
+                builder.append(DELIMITER);
+                builder.append(bufferedReader.readLine());
```

Comment:
You can use lineseparator and why you add delimiter before line?

Reply:
because later I split new array by this delimiter. I will replace this to lineseparator.